### PR TITLE
Scaling (StandardScaler, RobustScaler integration)

### PR DESCRIPTION
**Pull request by jtwinrelevanceai**

### CODE REVIEW

File: relevanceai/operations_new/ops.py [hunk 1]
```diff
@@ -515,3 +515,38 @@ def apply_transformers_pipeline(
...
+        alias: Optional[str] = None,
+        model_kwargs: Optional[dict] = None,
+        filters: Optional[list] = None,
+        batched: Optional[bool] = None,
+        chunksize: Optional[int] = None,
+    ):
+
+        from relevanceai.operations_new.scaling.ops import ScaleOps
+
+        model = "standard" if model is None else model
```
Comment (boba-and-beer) [inline relevanceai/operations_new/ops.py:532]: Why wouldn't you just do `model="standard"` ? Same with all the other args - if they're the default they should be set as default? We only do this for mutable objects like lists and dicts

**Review by boba-and-beer — APPROVED** [review]

File: relevanceai/operations_new/ops.py (hunk 1 above)
Comment (jtwinrelevanceai) [reply relevanceai/operations_new/ops.py:532]: True, was just keeping all optional args this way for consistency. Model IS optional

**Comment by boba-and-beer** [comment]
Remove a couple if statements here btw

**Comment by JackyKoh** [comment]
> Remove a couple if statements here btw

where should it be removed?

**Comment by JackyKoh** [comment]
There is still merge conflicts @spktrm

**Pull request merged**